src/cargo/core/compiler/unit.rs

```diff
@@ -82,6 +84,12 @@ pub struct UnitInner {
     /// [`BuildContext::extra_args_for`]: crate::core::compiler::build_context::BuildContext::extra_args_for
     /// [`TargetInfo.rustdocflags`]: crate::core::compiler::build_context::TargetInfo::rustdocflags
     pub rustdocflags: Arc<[String]>,
+    /// Build script override for the given library name.
+    ///
+    /// Any package with a `links` value for the given library name will skip
+    /// running its build script and instead use the given output from the
+    /// config file.
+    pub links_overrides: Arc<BTreeMap<String, BuildOutput>>,
```

Comment:
Just realized that it should be possible to use `Rc` instead of `Arc`, for all these usages.

Reply:
Huh, cargo check passes with that change at least. I think I must have just assumed they would cross a thread boundary somewhere. Will fix.